### PR TITLE
catalog--solution

### PR DESCRIPTION
**Pull request by maksym-kostetskyi**
https://maksym-kostetskyi.github.io/layout_catalog
https://maksym-kostetskyi.github.io/layout_catalog/report/html_report

### CODE REVIEW

File: src/styles/card.scss [hunk 1]
```diff
@@ -0,0 +1,99 @@
+p {
+  margin: 0;
+}
```
Comment (SanyaBratashchuk) [inline src/styles/card.scss:3]: It is not a part of card block

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,17 @@
+* {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
```
Comment (SanyaBratashchuk) [inline src/styles/main.scss:4]: Do not apply any rules to the `*` selector except `box-sizing`

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+}
+
+title {
+  display: none;
+}
```
Comment (SanyaBratashchuk) [inline src/styles/main.scss:9]: why do you need it?

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/styles/main.scss (hunk 3 above)
Comment (maksym-kostetskyi) [reply src/styles/main.scss]: I had an issue with the title displayed at the top of the page, and tests also were counting it and failing, and as it is necessary I just use display: none; now an issue seems to be solved, new commit without this code

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
it shouldn't be broken if I add more cards 
image

File: src/styles/catalog.scss [hunk 4]
```diff
@@ -0,0 +1,22 @@
+.catalog {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  padding: 50px 0;
+  justify-content: center;
```
Comment (SanyaBratashchuk) [inline src/styles/catalog.scss:6]:
```suggestion
```

File: src/styles/header.scss [hunk 5]
```diff
@@ -0,0 +1,95 @@
+.nav {
```
Comment (SanyaBratashchuk) [inline src/styles/header.scss:1]: why nav block is in header.scss file? Fix it

File: src/index.html [hunk 6]
```diff
@@ -1,12 +1,510 @@
...
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <!-- #region nav -->
+    <header class="nav">
```
Comment (SanyaBratashchuk) [inline src/index.html:20]:
```suggestion
    <header class="nav">
```
why header has a nav classname?

File: src/index.html [hunk 7]
```diff
@@ -1,12 +1,510 @@
...
+      <a href="#home" class="nav__logo-link">
+        <img
+          src="images/logo.png"
+          alt="MOYO logo"
+          class="nav__logo"
+        >
+      </a>
+
+      <nav class="nav__bar">
```
Comment (SanyaBratashchuk) [inline src/index.html:29]:
```suggestion
      <nav class="nav">
```

File: src/index.html (hunk 6 above)
Comment (SanyaBratashchuk) [inline src/index.html:20]:
```suggestion
    <header class="header">
```

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/styles/card.scss [hunk 8]
```diff
@@ -0,0 +1,100 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
```
Comment (alexandra-protyanova) [inline src/styles/card.scss:4]: do not set width directly to bem blocks, use bem mixes instead

File: src/styles/card.scss [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+    width: 100%;
+
+    font-weight: 400;
+    font-size: 10px;
+    text-align: right;
+  }
+
+  &__price {
+    display: flex;
+    flex-direction: row;
```
Comment (alexandra-protyanova) [inline src/styles/card.scss:54]: it is default value

File: src/styles/catalog.scss [hunk 10]
```diff
@@ -0,0 +1,21 @@
+.catalog {
+  display: flex;
+  flex-direction: row;
```
Comment (alexandra-protyanova) [inline src/styles/catalog.scss:3]: delete this property

File: src/styles/header.scss [hunk 11]
```diff
@@ -0,0 +1,17 @@
+.header {
+  display: flex;
+  align-items: center;
+  flex-direction: row;
```
Comment (alexandra-protyanova) [inline src/styles/header.scss:4]: here as well

File: src/styles/main.scss [hunk 12]
```diff
@@ -1,3 +1,23 @@
...
+
+p {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+}
+
```
Comment (alexandra-protyanova) [inline src/styles/main.scss:23]: add a blank line in the end of file

File: src/styles/nav.scss [hunk 13]
```diff
@@ -0,0 +1,78 @@
+.nav {
+  position: relative;
+
+  display: flex;
+  align-items: center;
+  flex-direction: row;
```
Comment (alexandra-protyanova) [inline src/styles/nav.scss:6]: why do u use it everywhere?

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]

File: src/styles/card.scss [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+    color: #616070;
+    font-weight: 400;
+    font-size: 10px;
+  }
+
+  &__stars {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-bottom: 24px;
```
Comment (ivanchuk-yurii) [inline src/styles/card.scss:39]: it's better to place `margin` on top before `display` as in two previous blocks

File: src/styles/nav.scss [hunk 15]
```diff
@@ -0,0 +1,76 @@
...
+        position: absolute;
+        bottom: 0;
+        left: 0;
+        right: 0;
+
+        border-top: 2px solid #00acdc;
+        border-bottom: 2px solid #00acdc;
+        border-radius: 2px;
+
+        content: "";
```
Comment (ivanchuk-yurii) [inline src/styles/nav.scss:72]: `content` should be on the beginning

**Review by ivanchuk-yurii — APPROVED** [review]

**Pull request closed without merging**